templates/primary-secondary/primary-secondary.js

```diff
@@ -70,6 +75,10 @@ class TemplatePrimarySecondary extends LitElement {
 			:host([primary-overflow="hidden"]) main {
 				overflow: hidden;
 			}
+			:host([background-shading="primary"]) main,
```

Comment:
If an internal class is applied to the `div` with `class="d2l-template-primary-secondary-content"`, then it's possible to avoid reflecting/sprouting `none` on the public API.

Reply:
We could also just not set a default value, so nothing should sprout.

Reply:
I left the default value but moved the attribute to an inner `div` and disabled `reflect` so the interface should be more consistent with native elements.